third_party/terraform/tests/resource_bigquery_connection_test.go.erb

```diff
@@ -0,0 +1,100 @@
+<% autogen_exception -%>
```

Comment:
Why does this file need the exception?

Reply:
blindly cargoculted, all the .erb test files seem to have it at the top, thoughts?